packages/batch_jaro_winkler/batch_jaro_winkler.0.1.1/opam

```diff
@@ -11,7 +11,7 @@ This project gets its performance from the pre-calculation of an optimized
 model in advance of the actual runtime calculations. Supports any encoding.
 """
 build: [
-  ["dune" "build" "-p" name "-j" jobs "--root=./ocaml/lib"]
+  ["dune" "build" "-j" jobs "--root=./ocaml/lib"]
```

Comment:
Keep the p, it includes also the release flag and target the specific named package only

Reply:
You can ignore the warning in the lint step

Reply:
Hi there, thanks for the feedback. Sadly, `-p` and `--root` seem to not be compatible. See here: https://opam.ci.ocaml.org/github/ocaml/opam-repository/commit/782baf0b2e7235b4486e5c6e72286f0c4db38f17/variant/compilers,4.02,batch_jaro_winkler.0.1.1. `# dune: Cannot use --root and -p simultaneously`. I have a hard time getting the linter to find the `dune-project` file: `The package seems to use dune but the dune-project file is missing.`. I tried to include it at the root, but then other things fail. Do you have an idea of what I could do?

Reply:
Sorry I didn't see your message about the lint step before. Nice! So do you prefer using `-p` or `--root`?

Reply:
Then do what `-p` does (https://dune.readthedocs.io/en/stable/opam.html) adding
`--only-packages pkg --profile release --default-target @install`
that line, or using `-p` and building in the right folder with [ "cd" "ocaml/lib"] before the dune invocation

Reply:
Thanks! It finally passed when using your suggestion, except for the lint.